co-simulation/fed/mosaic-carma/src/main/java/org/eclipse/mosaic/fed/carma/ambassador/CarmaMessageAmbassador.java

```diff
@@ -36,34 +36,35 @@ public class CarmaMessageAmbassador extends CommonMessageAmbassador<CarmaInstanc
     private Thread registrationRxBackgroundThread;
     private CarmaV2xMessageReceiver v2xMessageReceiver;
     private Thread v2xRxBackgroundThread;
-    private CarmaInstanceManager carmaInstanceManager;
+    private static CarmaInstanceManager carmaInstanceManager = new CarmaInstanceManager();
```

Comment:
Static is a weird access modifier for this. Is there a case where we need to access this carmaInstanceManager instance variable w/o actually having a constructed instance of the CarmaMessageAmbassador? That'd strike me as kind of off without further context, so I'd be curious to see where that arises if so.